metadata.yaml

```diff
@@ -0,0 +1,714 @@
+id: "terraform-ibm-modules/satellite-gcp/ibm/1.0.0"
+name: "satellite-gcp"
+namespace: "terraform-ibm-modules"
+version: "1.0.0"
+providers: ["ibm", "gcp"]
```

Comment:
Looks like we use 3 providers

provider "google" {
  project     = var.gcp_project
  region      = var.gcp_region
  credentials = var.gcp_credentials
}

provider "google-beta" {
  project     = var.gcp_project
  region      = var.gcp_region
  credentials = var.gcp_credentials
}

provider "ibm" {
  ibmcloud_api_key = var.ibmcloud_api_key
}

Reply:
Fixed